Orange/widgets/data/owsave.py

```diff
@@ -175,14 +180,18 @@ def save_file(self):
                     os.path.join(
                         self.last_dir,
                         self.basename + self.type_ext + self.compress_ext),
-                    self.data)
+                    self.data, self.with_annotations,
```

Comment:
This breaks compatibility with existing additional writers (if there are any!) that don't accept this argument. Writers that do not support it, should be called without it (see the general comment).